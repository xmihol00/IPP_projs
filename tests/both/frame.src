.IPPcode21

DEFVAR GF@x

CREATEFRAME
DEFVAR TF@y
ADD TF@y int@10 int@20
PUSHFRAME
WRITE LF@y
WRITE  string@\010
POPFRAME

SUB TF@y TF@y TF@y
WRITE TF@y
WRITE  string@\010
PUSHFRAME
MUL LF@y int@2 int@9
WRITE LF@y
WRITE  string@\010
