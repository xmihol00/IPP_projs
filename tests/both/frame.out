30
0
18
